src/components/App.css

```
@@ -11,3 +11,17 @@ h3, .cerise #content h3 {
 .pagination .disabled, .pagination .disabled a {
   color: #999 !important;
 }
+
+#secondary-nav a {
+  display: flex !important;
+  align-items: center;
+}
+
+.pageIcon {
```

```suggestion
.page-icon {
```
class name convention is kebab-case